src/main/resources/lib/menu/graphql.js

```diff
@@ -0,0 +1,107 @@
+const libs = {
+  graphql: require("/lib/graphql"),
+};
+
+function createGraphQLMenuTree(context) {
+  return context.schemaGenerator.createObjectType({
```

Comment:
`context.schemaGenerator` is a part of `lib-guillotine` not `lib-graphql`. Also if you assume that should be available in guillotine `context` then object type has to be added to `context.types`, for instance:
```
context.types.menu = ...
```
Also `lib-guillotine` uses custom `lib-graphql` which overrides for instance, `createObjectType` function to use `creationCallback`.

In other words, the `app` has to include `lib-guillotine` in dependencies.

Reply:
`lib-graphql` exposes `newSchemaGenerator` function

Reply:
@anatol-sialitski Would it make more sense to take `schemaGenerator` as a parameter, instead of `context`? Then the developer can pass in a schema generator from either `lib-graphql` or `lib-guillotine` right?

Reply:
@tajakobsen I wanted to highlight that will not enough to use only `lib-graphql`. You need to include `lib-guillotine`, because `context`, `context.schemaGenerator`, `context.uniqueName` and `creationCallbacks` are the features of the lib-guillotine.